src/Tasks/Microsoft.NET.Build.Tasks/ResolveAppHosts.cs

```diff
@@ -186,12 +191,26 @@ protected override void ExecuteCore()
                 if (wasInGraph)
                 {
                     //  NETSDK1084: There was no app host for available for the specified RuntimeIdentifier '{0}'.
-                    Log.LogError(Strings.NoAppHostAvailable, runtimeIdentifier);
+                    if (errorIfNotFound)
```

Comment:
why this change?

In general i don't like a flag to not make error to message. It is like a flag to control throw or not.